### ISSUE TITLE
[Hint][Flexbox] Report Shrink-to-fit containers with flex-flow: column wrap do not contain their items

### ISSUE THREAD

**Issue by captainbrosset** [webhintio/hint#4068]
_(from https://github.com/philipwalton/flexbugs#flexbug-14)_

If you `float` a flex container, use `inline-flex`, or absolutely position it, the size of the container becomes determined by its content (a.k.a shrink-to-fit).

When using `flex-flow: column wrap`, some browsers do not properly size the container based on its content, and there is unwanted overflow. Demo [14.1.a](https://philipwalton.github.io/flexbugs/14.1.a-bug.html) shows an example of this.

Test case: https://philipwalton.github.io/flexbugs/14.1.a-bug.html

Bug (in Edge)
<img width="449" alt="bug" src="https://user-images.githubusercontent.com/1152698/95330694-3795ee00-08a9-11eb-8fc9-32d8cc579c44.png">

Most (all?) browsers seem to be affected, and they all have issues on file to address this. But it be word considering whether webhint should report this if it isn't really a compatibility problem (since all browsers do the same (albeit broken) thing).

**Comment by antross** [comment]
Yeah, if browsers are already consistent this probably becomes more interesting if/when at least one browser has fixed the bug.
